migration-guides/v6.0.0.md

```diff
@@ -77,3 +78,14 @@ Any `<RangeInput />` with a `hideLabel` prop should be updated to use `hideRange
 // After
 <RangeInput input={rangeInputProps} hideRangeValue={true} />
 ```
+#### 6. The tag on `<Spinner />` now uses the classname `spinner` in place of an id
+
+`<Spinner />` utilizes a classname `spinner` in order to accomodate views where more than one spinners are necessary.
+
+```jsx
+// Before
+<div id="spinner" { ...filterInvalidDOMProps(props) } />
+
+// After
+<div className={classnames('spinner', props.className)} />
```

Comment:
This guide is focused on the change that a developer has to make to their existing code, not the actual code change that was made. The breaking change here is the styling change. If a dev updated their project to use this version of `lp-components`, the actually spinner animation wouldn't show up until they modified their scss files (i.e., replacing any reference of `#spinner` with `.spinner`)